PhysicsOverlayInstructions.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  PhysicsOverlayInstructions.swift
+//  orpheum-game
+//
+//  Created by Yichen Yao on 4/2/17.
+//
+//
+import SpriteKit
+
+class SnowballGamePhysics: SKShapeNode {
```

Comment:
Let's make the name of this class a bit more informative. Something that let's us know it explains the physics behind the game.